import_processor/readme/USAGE.rst

```diff
@@ -0,0 +1,31 @@
+This module provides a set of pre-defined functions and wizards to facilitate importing data into Odoo.
+
+The module offers predefined variables that can be used when writing code snippets in the view.
+
+Note: Only pre-defined variables can be used in code snippets.
```

Comment:
The pre-defined variables are quite powerful (which is the reason only admin can edit/create those). Here it sounds very limited which isn't the case. All the above lines have nothing to do with the usage. Usage would be: how to configure/set up and how to use (Favorite menu on the models)